rhel8/CMakeLists.txt

```diff
@@ -23,6 +22,11 @@ ssg_build_html_anssirefs_table(${PRODUCT} "bp28_intermediary")
 ssg_build_html_anssirefs_table(${PRODUCT} "bp28_enhanced")
 ssg_build_html_anssirefs_table(${PRODUCT} "bp28_high")
 
+ssg_build_html_cisrefs_table(${PRODUCT} "cis_l1_server")
```

Comment:
Up to the RHEL folks, but I'd maybe consider spelling out `level` to make it consistent with the proposed Ubuntu #6416 PR?

Reply:
I couldn't really decide either way when choosing the naming, but this seems sensible enough and I think there's a definite benefit to consistency. I've added a commit to switch to "level".